frontend/src/main/components/PersonalSchedule/PersonalScheduleForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+import { quarterRange } from "../../utils/quarterUtilities";
+
+function PersonalScheduleForm({
+  initialPersonalSchedule,
+  submitAction,
+  currentUser,
+  buttonLabel = "Create",
+}) {
+  // Stryker disable all
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    setValue,
+  } = useForm({ defaultValues: initialPersonalSchedule || {} });
+  // Stryker enable all
+
+  const navigate = useNavigate();
+  const [quarter, setQuarter] = useState("20213");
```

Comment:
At the moment, the quarter 20213 is always used if you don't change the quarter in the SingleQuarterDropdown selector.

The default quarter that will be returned from SingleQuarterDropdown is this:
```
  const [quarterState, setQuarterState] = useState(
    // Stryker disable next-line all : not sure how to test/mock local storage
    localSearchQuarter || quarters[0].yyyyq
    );
 ```   

I think you need to do the same thing.   That is, your default will always be wrong if you hard code it to anything.

I suspect that it needs to match the default being used in SingleQuarterDropdown if it's going to work properly.

Which comes from localstate:
```
const localSearchQuarter = localStorage.getItem(controlId);
```
Where the controlId is the one you pass in:
```
          controlId={"PersonalScheduleForm-quarter"}
```

I think the way to fix the bug is to align the defaults.

Otherwise, whatever quarter you hard code on the line:
```
  const [quarter, setQuarter] = useState("20213");
```
that's the quarter that will get chosen if the user doesn't change anything in the quarter selector.

And if they don't change anything the quarter selector, what they WANT is the default quarter, which is
the one that is in localstorage under key "PersonalScheduleForm-quarter", or if there isn't one there, its the
first thing in the list of quarters passed in to the SingleQuarterDropdown.

I could  be wrong.